tiledb/sm/storage_manager/storage_manager.h

```diff
@@ -687,6 +687,19 @@ class StorageManager {
   Status get_latest_array_schema_uri(
       const URI& array_uri, URI* schema_uri) const;
 
+  /**
+   * Loads the schema of a schema uri from persistent storage into memory.
+   *
+   * @param array_schema_uri The URI path of the array.
```

Comment:
Make sure the argument name is the same.

Reply:
fixed